bastion_host.tf

```diff
@@ -0,0 +1,16 @@
+resource "aws_instance" "Bastion_Host" {
+  ami           = "ami-068257025f72f470d"
```

Comment:
Could you use a data source instead for looking for the ami ?

Reply:
ok, we will make the required changes.